supporter-product-data/src/main/scala/com/gu/services/SqsService.scala

```diff
@@ -0,0 +1,57 @@
+package com.gu.services
+
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.sqs.AmazonSQSAsyncClientBuilder
+import com.amazonaws.services.sqs.model.{
+  SendMessageBatchRequest,
+  SendMessageBatchRequestEntry,
+  SendMessageRequest,
+  SendMessageResult,
+}
+import com.gu.aws.{AwsAsync, CredentialsProvider}
+import com.gu.monitoring.SafeLogger
+import com.gu.monitoring.SafeLogger._
+import com.gu.supporterdata.model.{ContributionAmount, Stage, SupporterRatePlanItem}
+import io.circe.Encoder
+import io.circe.generic.semiauto.deriveEncoder
+import io.circe.syntax.EncoderOps
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.jdk.CollectionConverters._
+
+class SqsService(queueName: String)(implicit val executionContext: ExecutionContext) {
+  implicit val encoder: Encoder[SupporterRatePlanItem] = deriveEncoder
```

Comment:
does some of this belong in the companion object?

Reply:
`SupporterRatePlanItem` is defined in a project which doesn't  have a dependency on Circe so to avoid adding it we need to define codecs here.

Reply:
nice, I'm heartened that dependencies (aka liabilities!) are a consideration for code structure